vb_suite/groupby.py

```diff
@@ -559,3 +559,6 @@ def inject_bmark_into_globals(bmark):
     for func_name in no_arg_func_list:
         bmark = make_large_ngroups_bmark(ngroups, func_name)
         inject_bmark_into_globals(bmark)
+
+# avoid bmark to be collected as Benchmark object
+del bmark
```

Comment:
without this fix, I got following error:
```
$ rm vb_suite/benchmarks.db
$ ./test_perf.sh -b master -t HEAD -r groupby
...
2015-01-22 20:47:50,103 [INFO  ] Writing new benchmark groupby_ngroups_100_value_counts, b88a3ca315564ed6afdd935e4d2dc8ad (runner.py:153)
2015-01-22 20:47:50,105 [INFO  ] Writing new benchmark groupby_ngroups_10000_value_counts, f7a6e1f815904619a0428842d3fa2ffe (runner.py:153)
2015-01-22 20:47:50,107 [INFO  ] Writing new benchmark groupby_ngroups_10000_sum, 973f817c1bd057ddc5476e5af7667ed6 (runner.py:153)
2015-01-22 20:47:50,118 [INFO  ] Writing new benchmark groupby_ngroups_10000_value_counts, f7a6e1f815904619a0428842d3fa2ffe (runner.py:153)
...
sqlalchemy.exc.IntegrityError: (IntegrityError) UNIQUE constraint failed: benchmarks.checksum u'INSERT INTO benchmarks (checksum, name, description) VALUES (?, ?, ?)' ('f7a6e1f815904619a0428842d3fa2ffe', 'groupby_ngroups_10000_value_counts', None)
```

This is because ``suite`` module collects all ``Benchmark`` objects in ``groupby`` module, while temporary variable ``bmark`` presents outside the for loop.